packages/terraform-modules-public/testnet/modules/attestation-service/variables.tf

```diff
@@ -0,0 +1,100 @@
+variable attestation_service_count {
+  type        = number
+  description = "Number of Attestation Service to deploy"
+}
+
+variable celo_env {
+  type        = string
+  description = "Name of the testnet Celo environment"
+}
+
+variable gcloud_region {
+  type        = string
+  description = "Name of the Google Cloud region to use"
+}
+
+variable instance_type {
+  description = "The instance type"
+  type        = string
+  default = "n1-standard-1"
+}
+
+variable attestation_service_docker_image_repository {
+  type        = string
+  description = "The docker image repository for the attestation service"
+}
+
+variable attestation_service_docker_image_tag {
+  type        = string
+  description = "The docker image tag for the attestation service"
+}
+
+variable db_username {
+  type        = string
+  description = "The User for the database"
+}
+
+variable db_password {
+  type        = string
+  description = "The password for the database"
+}
+
+variable network_name {
+  type        = string
+  description = "The name of the network to use"
+}
+
+variable account_address {
+  type        = list(string)
+  description = "The account address for signing the attestations. Must be the address of the associated validator"
+}
+
+variable attestation_key {
+  type        = list(string)
+  description = "The account private key for signing the attestations. Must be the private key of the associated validator"
```

Comment:
Answering my question, this can be the "attestation signer key" according to here https://docs.celo.org/getting-started/baklava-testnet/running-a-validator#account-and-signer-keys